modules/apps/jaggery/emm/src/emm/modules/device.js

```diff
@@ -319,16 +319,19 @@ deviceModule = function () {
     publicMethods.getLicense = function (deviceType, languageCode) {
         var url;
         var license;
-        if(deviceType != null && deviceType != undefined && deviceType == "windows"){
+        if (deviceType != null && deviceType != undefined && deviceType == "windows") {
```

Comment:
you can simply use - if (deviceType == "windows"), and use a constant for accessing deviceType